### PR TITLE
Deprecate schedule calendar page and related ajax calls

### PR DESCRIPTION
**Pull request by jamiesjc**
Several reasons for this change:
1. Schedule calendar page is not exposed and used by anyone. 
2. When loading schedule calendar page, it will trigger both loadFlow and loadHistory ajax calls. They will create .cache file and store in web server /cache folder, which will take unnecessary disk space. And the .cache files are not cleaned up properly. 
3. This is part of the steps to simplify updaterThread logic.

### CODE REVIEW

File: azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java [hunk 1]
```diff
@@ -1337,14 +1336,6 @@ public void run() {
               }
             }
 
-            for (ExecutableFlow flow : finishedFlows) {
-              if (flow.getScheduleId() >= 0
-                  && flow.getStatus() == Status.SUCCEEDED) {
-                ScheduleStatisticManager.invalidateCache(flow.getScheduleId(),
-                    cacheDir);
```
Comment (jamiesjc) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: In updaterThread, the original intention of this code was to delete the .cache file for the flow when it succeeded. However, there was some bug and flow.getScheduleId() was always -1 and never updated correctly. So .cache file was never deleted once created. So I removed this code and simplified the logic in updaterThread.

Comment (kunkun-tang) [inline azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java]: Great Explanation!

**Comment by HappyRay** [comment]
Great to see "dead code" gone.

Thanks!

**Comment by wyukawa** [comment]
@jamiesjc 

Oh, I use loadFlow API because I want to get all schedule data.

https://github.com/wyukawa/eboshi/blob/master/eboshi/schedule.py#L18

Is there alternative API?

**Comment by wyukawa** [comment]
send pull request https://github.com/azkaban/azkaban/pull/1219

**Pull request merged**